src/sagas/repository.js

```diff
@@ -28,9 +28,31 @@ export function* createDataset({ payload }) {
   }
 }
 
+export function* enumerateDatasets({ payload }) {
+  try {
+    const response = yield call(
+      axios.get,
+      '/api/repository/v1/datasets?offset=0&limit=10',
+      payload,
```

Comment:
Isn't payload empty for a GET? I would prefer instead of hardcoding the offset and limit into the url that we accept them as options in the arguments.

```js
export function* enumerateDatasets({ payload }) {
  try {
    const response = yield call(
      axios.get,
      '/api/repository/v1/datasets'
      { params: payload }
    );
    ...
```
This will let us include things like sort and custom offset/limits later on for when we need to implement server-side sorting, paging, etc.